### PR TITLE
fixes jar, adds ec to default role

### PR DESCRIPTION
**Pull request by tom-smith-okta**
Problem Statement
-----------------
The default jar included in the repo resulted in an unusable UI (role names were listed incorrectly and could not be successfully selected).

Solution
--------
Built a new jar file for inclusion in the /out dir. Also improved error-checking for reading a default role from the config.properties file (now checks for empty string in addition to null).

Before:
before

After:
after

### CODE REVIEW

File: src/main/java/com/okta/tools/OktaAwsCliAssumeRole.java [hunk 1]
```diff
@@ -312,8 +312,8 @@ private AssumeRoleWithSAMLRequest chooseAwsRoleToAssume(String samlResponse) thr
                 i++;
             }
         }
-        if (awsRoleToAssume != null && j == -1) {
-            System.out.println("No match for role " + awsRoleToAssume);
+        if ((awsRoleToAssume != null && !awsRoleToAssume.isEmpty()) && j == -1) {
+            System.out.println("\ncould not find the role " + awsRoleToAssume + "\n specified in the config.properties file.");
```
Comment (jmelberg-okta) [inline src/main/java/com/okta/tools/OktaAwsCliAssumeRole.java]: nit: We should capitalize this error message + keep the error string as one line. So we can remove the `\n` newline after `awsRoleToAssume`

Comment (AlainODea) [reply src/main/java/com/okta/tools/OktaAwsCliAssumeRole.java]: **awsRoleToAssume** isn't necessarily from the config.properties file

**Comment by AlainODea** [comment]
I don't think that **oktaawscli.jar** should be committed. I think that's a historical accident since Git is not intended as an artifact repository. It really bloats the repo and I think it should be handled externally.

File: src/main/java/com/okta/tools/OktaAwsCliAssumeRole.java (hunk 1 above)
Comment (AlainODea) [reply src/main/java/com/okta/tools/OktaAwsCliAssumeRole.java]: I disagree with making this logging change.

**Comment by bdemers** [comment]
+100, don't commit artifacts!

**Comment by rdegges** [comment]
I'm going to allow it. When @mraible gets back he can always fix the build pipeline if he's so inclined. Right now getting stuff fixed quickly even if we're committing binaries is more useful since it's been that way since inception.

Definitely not the right thing to do in the long run, but atm since we don't have a maintainer this is progress!

Thanks @tom-smith-okta !! <33

**Pull request merged**